src/plutus_gen/mod.rs

```diff
@@ -72,6 +75,14 @@ where
         &circuit_representation,
     )
     .map_err(|e| e.to_string())?;
+    emit_vk_code_aiken(
```

Comment:
better create a separate `generate_aiken_verifier` and put it there to keep things ordered since the beginning